src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -1,13 +1,13 @@
 package mesosphere.marathon
 package core.instance
 
-import java.util.{ Base64, UUID }
+import java.util.{Base64, UUID}
```

Comment:
Formatting :)

Reply:
Which one do we use? I've seen both.

Reply:
I think the formatting will be automatically fixed for you when you compile, right? :)